static/elements/chromedash-guide-editall-page.js

```diff
@@ -142,15 +141,9 @@ export class ChromedashGuideEditallPage extends LitElement {
         `)}
 
         <section class="final_buttons">
-          <chromedash-form-table>
-            <chromedash-form-field>
-              <span slot="field">
-                <input class="button" type="submit" value="Submit">
-                <button id="cancel-button" type="reset"
-                  @click=${this.handleCancelClick}>Cancel</button>
-              </span>
-            </chromedash-form-field>
-          </chromedash-form-table>
+          <input class="button" type="submit" value="Submit">
```

Comment:
In this or a follow-up PR, it would be good to convert these to sl-buttons.

Reply:
I'll do these conversions in a follow-up PR.